tox.ini

```diff
@@ -19,13 +20,31 @@ commands =
     flake8 src tests
 
 [pytest]
+addopts = --cov=nidm --no-cov-on-fail
 filterwarnings =
     error
     # <https://github.com/sensein/etelemetry-client/pull/44>
     ignore:.*pkg_resources:DeprecationWarning
     # <https://github.com/PythonCharmers/python-future/issues/246>
     ignore:the imp module is deprecated:DeprecationWarning
 
+[coverage:run]
+branch = True
+parallel = True
+
+[coverage:paths]
+source =
+    src
+    .tox/**/site-packages
```

Comment:
why do we need both `src` and  `.tox` paths here? i.e. which is one is really in use or really both while running tox testing?

Reply:
The point of this config setting is to get `coverage` to treat paths installed under `.tox` as though they were actually the equivalent paths under `src`.

Reply:
thanks, not yet sure how/why that would do it but I will just trust you on this ;)